product_multi_category/README.rst

```diff
@@ -38,7 +31,7 @@ help us smashing it by providing a detailed and welcomed `feedback
 <https://github.com/OCA/
 product-attribute/issues/new?body=module:%20
 product_multi_category%0Aversion:%20
-9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Check current README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst. I mean the short link.